recipes/cylc-rose/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cylc-rose" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 931b239267ae05deeabbe9142d92fab3a8fad79f3da52477eac5b0b4500ed6d7
+
+build:
+  number: 0
+  skip: True  # [py<37 or not linux]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
```suggestion
    - noarch: python
```

Reply:
The cylc-flow dependency is Linux only, how do I disable the Windows and MacOS builds with `noarch: python`?

Reply:
If it doesn't work on anything but Linux, then your skip above is appropriate. You should still remove the `build` section here, though, since you don't actually use the C compiler.